executor/compiler.go

```diff
@@ -50,12 +50,18 @@ func (c *Compiler) Compile(ctx context.Context, stmtNode ast.StmtNode) (*ExecStm
 		return nil, errors.Trace(err)
 	}
 
+	ReadOnlyCheckStmt := stmtNode
```

Comment:
For local variables, we usually do not need to capitalize the name.

Reply:
ok